src/components/modal/v2/parts/Donut.jsx

```diff
@@ -0,0 +1,89 @@
+/** @jsx h */
+/* eslint-disable react/no-unknown-property */
+import { h } from 'preact';
+
+const generateDonutData = (currentNum = 0, numOfPayments = 4) => {
+    return {
+        percentage: (currentNum / numOfPayments) * 100
+    };
+};
+
+const Donut = ({
+    strokeWidth = 3,
+    strokeLinecap = 'round',
+    cx = 21,
+    cy = 21,
+    radius = 15.91549430918954,
+    viewBox = `0 0 ${2 * cx} ${2 * cy}`,
+    style = { fontSize: '0.375rem' },
+    children,
+    segmentStrokeWidth = 4,
+    currentNum = 0,
```

Comment:
```suggestion
    currentNum = 0,
    numOfPayments = 4
```
We probably need to pass in the above so that whatever number of payments we pass into the Donuts component is correctly generated (i.e. for UK's Pay in 3)

Reply:
Made this update